src/singletons/Settings.hpp

```diff
@@ -125,6 +125,8 @@ class Settings : public ABSettings, public ConcurrentSettings
         "/behaviour/autocompletion/smallStreamerLimit", 1000};
     BoolSetting prefixOnlyEmoteCompletion = {
         "/behaviour/autocompletion/prefixOnlyCompletion", true};
+    BoolSetting userCompletionOnlyWithAt = {
+        "/behaviour/autocompletion/userCompletionOnlyWithAt", true};
```

Comment:
This changes the default behaviour - we should probably default this to `false`

Reply:
Changed